processor/src/main/java/org/mapstruct/ap/spi/MethodType.java

```diff
@@ -43,5 +43,10 @@
     /**
      * Any method which is neither a JavaBeans getter, setter nor an adder method.
      */
-    OTHER;
+    OTHER,
+
+    /**
+     * A JavaBeans hasser method, e.g. {@code public String hasName()}.
+     */
+    HASSER;
```

Comment:
I'm still no fan of "HASSER" :) How about "PRESENCE_CHECK" or "IS_VALUE_SET_CHECKER" or so?

Also better not refer to JavaBeans as there is no such concept in the spec.